Git/commit.sublime-syntax

```diff
@@ -0,0 +1,84 @@
+%YAML 1.2
+---
+# Highlight regular git commits, merge commits, and tags.
+
+name: Git Commit
+file_extensions:
+  - COMMIT_EDITMSG
+  - MERGE_MSG
+  - TAG_EDITMSG
+  - syntax_test_git_commit
+  - syntax_test_git_merge
+  - syntax_test_git_tag
```

Comment:
These syntax_test_* file extensions are temporary, right? I believe ST picks up the syntax automatically for syntax tests anyway, I think you can remove these even when developing.

Reply:
I'm happy to remove these! Sublime will open the syntax tests as plain text, but it's easy to switch the syntax. The problem is that the `COMMIT_EDITMSG`, `MERGE_MSG`, and `TAG_EDITMSG` files don't have extensions and don't have a predictable first line of text for `first_line_match`.

I think you're totally right that these shouldn't be in the shipped version, though.

Reply:
>I believe ST picks up the syntax automatically for syntax tests anyway

That's a feature of PackageDev, actually.